rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-10-28
+nightly-2022-02-23
```

Comment:
This possibly implies a chance in the whole stack. Example: https://github.com/dusk-network/Poseidon252/blob/f6c1e914956d115858cc796fbaaa3063e763de56/rust-toolchain

Reply:
No, since the rust-toolchain is per repo is not a problem.
That means if we build poseidon alone it uses nightly-2021-10-28 but if we build poseidon as dependency of rusk it will uses the 2022-02-23.

So if rusk pass everything should be good.
I also checked that this is the last toolchain that contains all the target and components we're using.

Reply:
Unless some bug that's not the case; we have already more than one version of the toolchain in our stack, see:

[dusk-abi](https://github.com/dusk-network/dusk-abi) (no toolchains pecified), [dusk-bytes](https://github.com/dusk-network/dusk-bytes/blob/main/rust-toolchain) (`nightly-2021-06-06`), etc.

The reason why we opted to have a `rust-toolchain` in each of our repos it's because we can decide to change the toolchain independently one from the other. Basically when `poseidon252` is built from its own repo source code, it will uses the `rust-toolchain` specified there; but if it's used as dependency of `rusk` it will be compiled with `nightly-2022-02-23`.